redisson/src/main/java/org/redisson/client/handler/CommandPubSubDecoder.java

```diff
@@ -222,24 +223,16 @@ protected MultiDecoder<Object> messageDecoder(CommandData<Object, Object> data,
                 return null;
             }
             return commandData.getCommand().getReplayMultiDecoder();
-        } else if ("message".equals(command)) {
+        } else if (TYPE_MESSAGES.contains(command)) {
             byte[] channelName = (byte[]) parts.get(1);
             PubSubEntry entry = entries.get(new ChannelName(channelName));
             if (entry == null) {
                 return null;
             }
             return entry.getDecoder();
-        } else if ("pmessage".equals(command)) {
-            byte[] patternName = (byte[]) parts.get(1);
-            PubSubEntry entry = entries.get(new ChannelName(patternName));
-            if (entry == null) {
-                return null;
-            }
-            return entry.getDecoder();
         } else if ("pong".equals(command)) {
             return new ListObjectDecoder<>(0);
         }
-
```

Comment:
please retain only codec changes

Reply:
should I remove CommandPubSubDecoderTest?

Reply:
Sorry, I meant remove deletions of empty lines.

Reply:
Done, deletion reverted.